Gruntfile.js

```diff
@@ -3,8 +3,6 @@ module.exports = function (grunt) {
 
   require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
 
-  function init () {
-
     grunt.initConfig({
```

Comment:
Should we fix the indentation if we make this change?

Reply:
+1 done